src/graph/topo.cc

```diff
@@ -642,8 +642,7 @@ ncclResult_t scclCheckBufferBounds(int bufferType, int offset, int nInputChunks,
   return ncclSuccess;
 }
 
-ncclResult_t scclGetAlgoFromXMLAndSetComm(struct ncclComm* comm) {
-  char* str = getenv("SCCL_XML_FILE");
+ncclResult_t scclGetAlgoFromXMLAndSetComm(struct ncclComm* comm, const char* str) {
```

Comment:
So does this mean we can take multiple XML files?

Reply:
Not yet.